packages/abi-gen-wrappers/src/generated-wrappers/coordinator.ts

```diff
@@ -56,6 +56,62 @@ export class CoordinatorContract extends BaseContract {
             return result;
         },
     };
+    public getTransactionHash = {
```

Comment:
What are these changes?

Reply:
Capturing from Slack: I updated the `Coordinator.json` artifact in [this PR](https://github.com/0xProject/0x-monorepo/pull/1792/files#diff-220fa7bd7fd736b670332bab49179793) because it was outdated from `Coordinator.sol`, but didn't know about updating abi-gen-wrappers.

Reply:
Note that `contract-artifacts/CHANGELOG.json` should probably be updated with these changes.